---
plugins:
  dsg_mesh:
    type: MeshPlugin
    color_by_label: $(arg color_mesh_by_label)
    label_colormap: $(arg semantic_map_path)
  gt_regions:
    type: GtRegionPlugin
    gt_regions_filepath: $(arg gt_regions_filepath)
    draw_labels: true
    label_offset: 2.0
    fill_polygons: false
    use_boundary_color: true
    line_width: 0.20
    label_scale: 1.0
  # places_basis_points:
  #   type: BasisPointPlugin
  #   show_voxblox_connections: false
  #   draw_basis_points: true
  #   basis_point_scale: 0.1
  #   places_node_scale: 0.1
